### ISSUE TITLE
fedmsg "Register" button currently doesn't work

### ISSUE THREAD

**Issue by sochotnicky** [fedora-infra/mobile#25]
There's no error message or "wait" animation. It just silently does nothing :-) 

That's after adding the API key, previously the fedmsg notifications were showing (what I assume to be) all fedmsg messages.

**Comment by relrod** [comment]
Yeah, the whole fmn workflow in Mobile is very much in flux and needs to be smoothed out.

I was hoping to get a lot done with it this weekend, but I ended up getting sick have just been trying to rest. :-(

I think right now the registration process "works" for some definition of "works." There was never any code for getting notifications about fmn events, just for completing the registration process, which is where we stand right now.

So really this is two things:
- Implement the handling/rendering of fmn notifications
- Polish and document the registration process.

**Comment by sochotnicky** [comment]
OK, I guess I put the card before the horse in this case. But no worries, IRC bot works and I have android push notifications in my IRC client so it's OK (for now anyway)
